aten/src/ATen/native/cuda/UpSampleNearest1d.cu

```diff
@@ -22,7 +22,7 @@ __global__ void upsample_nearest1d_out_frame(
     size_t src_dim_w,
     size_t dst_dim_w,
     scalar_t* output) {
-  int dst_idx = blockIdx.x * blockDim.x + threadIdx.x;
+  size_t dst_idx = blockIdx.x * blockDim.x + threadIdx.x;
```

Comment:
why switch to `size_t`?

Reply:
I just reverted UpSampleNearest1/2/3d.cu because of some CI failures. 
It seems like this was changed in another PR, so by reverting to an older commit in this PR I lost the merge.

Reply:
(same for the other "ditto")